src/components/Dashboard/FilingHistoryList/LimitedRestorationFiling.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div v-if="filing">
+    <h4>Limited Restoration Period</h4>
+
+    <p class="limited-restoration-period">
+      The Company <strong>{{ filing.legalName }}</strong> was successfully
+      restored and active <strong>until {{ expiryDateFriendly }}</strong>
+      At the end of the limited restoration period, the company will be automatically dissolved.
+      If you require assistance to extend a limited restoration/reinstatement or wish to convert
+      your restoration from a limited period to a full restoration, please contact BC Registries staff:
+    </p>
+
+    <ContactInfo class="mt-4" />
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Prop } from 'vue-property-decorator'
+import { HistoryItemIF } from '@/interfaces'
+import { ContactInfo } from '@/components/common'
+import { DateMixin } from '@/mixins'
+
+@Component({
+  components: { ContactInfo },
+  mixins: [DateMixin]
+})
+export default class LimitedRestorationFiling extends Vue {
+  /** The subject filing. */
+  @Prop({ required: true }) readonly filing!: HistoryItemIF
+
+  /** The expiry datetime of the limited restoration filing. */
+  get expiryDateFriendly (): string {
+    return (this.formatYyyyMmDd(this.filing.expiryDate) || 'Unknown')
+  }
+}
+
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+
+.limited-restoration-period {
+  color: dimgrey;
```

Comment:
I believe we don't use this colour. Check theme.scss for a suitable grayX colour variable (same as other sub-component in FilingHistoryList).

Reply:
Hmm, I couldn't find another FilingHistory sub-component using `$gray#`, but `$gray7` seems like a close match to what the UI requested.  Let me know if you'd prefer something different.

Reply:
Sounds good to me.